test/Npgsql.Tests/Npgsql.Tests.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
```

Comment:
I'd rather not bring in the whole package just for this one test. If you can't figure out any other way to assert this, I'd be OK with just removing the test.

Reply:
Well, I could check Exception._message using Reflection, I guess, but the original issue is probably a JSON.NET bug, so there is no way to test for that without referencing JSON.NET.

Npgsql.PluginTests already indirectly depends on Newtonsoft.Json. Maybe the test can go there instead?

Reply:
OK, I ended up merging this as-is - if the added dependency every causes trouble we can revisit this.